### PR TITLE
Update replace() definition from @types/node-emoji

### PR DESCRIPTION
**Pull request by rimiti**
Hello,

This pull request fix the `replace` function definition.

Concerned line of the official repository: 
https://github.com/omnidan/node-emoji/blob/5975ad506a97465d4050f8895a02b79cccea8373/lib/emoji.js#L272

PS: If you are seeking a new maintainer, I'm in! 🚀

### CODE REVIEW

**Review by styu — APPROVED** [review]

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by DanielRosenwasser** [comment]
> If you are seeking a new maintainer, I'm in! rocket

It can be a burden as a heads up, but if you're interested, feel free to add yourself to the header comment at the top!

File: types/node-emoji/index.d.ts [hunk 1]
```diff
@@ -1362,4 +1363,4 @@ export function unemojify(str: string): string;
 export function find(emoji: string): Emoji;
 export function hasEmoji(str: string): boolean;
 export function strip(str: string): string;
-export function replace(str: string, callback: (emoji: Emoji) => string): string;
+export function replace(str: string, replacement: (emoji: Emoji) => string | string, cleanSpaces?: boolean): string;
```
[the remark this answers is not in the document]
Comment (rimiti) [reply types/node-emoji/index.d.ts]: Oh! Your right, I was distracted.
It's fixed.

**Comment by rimiti** [comment]
It can be merged.

**Comment by DanielRosenwasser** [comment]
Thank you!

**Pull request merged**